File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(s):
    p_count = 0
    y_count = 0
    s = s.lower()
    for i in range(len(s)):
        if s[i] == "p":
            p_count += 1
        elif s[i] == "y":
            y_count += 1
    if p_count == y_count:
        return True
    else:
        return False

File: Java/프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
	    s = s.toLowerCase();

	    int pCount = s.length() - s.replace("p", "").length();
	    int yCount = s.length() - s.replace("y", "").length();

	    return pCount == yCount;

    }
}